lib/modules/manager/pub/artifacts.spec.ts

```diff
@@ -0,0 +1,156 @@
+import _fs from 'fs-extra';
+import { join } from 'upath';
+import { envMock, mockExecAll } from '../../../../test/exec-util';
+import { mocked } from '../../../../test/util';
+import { GlobalConfig } from '../../../config/global';
+import type { RepoGlobalConfig } from '../../../config/types';
+import * as _env from '../../../util/exec/env';
+import type { UpdateArtifactsConfig } from '../types';
+import * as pub from './artifacts';
+
+jest.mock('fs-extra');
```

Comment:
don't mock fs-extra, checkout other manager

```suggestion
jest.mock('../../../util/fs');
```